lightning/src/util/events.rs

```diff
@@ -27,6 +28,47 @@ use prelude::*;
 use core::time::Duration;
 use core::ops::Deref;
 
+#[derive(Clone, Debug)]
+pub enum ClosureDescriptor {
+	ForceClosed,
+	UserInitiated,
+	CounterpartyInitiated,
+	CooperativeClosure,
+	UnknownOnchainCommitment,
```

Comment:
Hmm, maybe rename this "CommitmentTxBroadcasted" or so - "Unknown" feels strange as we probably do know *which* commitment it is.